Automattic/newspack-theme
# Removes priority menu functionality

### All Submissions:


### Changes proposed in this Pull Request:

This PR removes the 'priority' menu functionality that originated from Twenty Nineteen.

This is the functionality that adds the `...` icon at the end of the menu, when you add too many menu items for the space available:

![image](https://user-images.githubusercontent.com/177561/61085291-b66d5400-a3e4-11e9-9361-9e8ad9bd9a58.png)

It doesn't look like something we can replicate 1:1 with AMP, and with the increased complexity of the menus in this theme, it's probably best to simplify anyway :) 

### How to test the changes in this Pull Request:

1. Add a very long menu to the 'Primary' location.
2. Make sure the 'more' link is displaying
3. Apply the PR and run `npm run build`
4. Confirm that the 'more' link is no longer there, and the menu wraps to two lines.

Note: This should look better if applied after #97 is merged, but it may still need design adjustment. 

### Other information:
